packages/charts/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -1010,43 +1017,8 @@ export class CartesianChartBase
   };
 
   private _calculateChartMinWidth = (): number => {
-    let labelWidth = 10; // Total padding on the left and right sides of the label
```

Comment:
>labelWidth = 10 [](http://example.com/codeflow?start=8&length=15)

this value not needed anymore?

Reply:
It is now defined here: https://github.com/microsoft/fluentui/blob/65b1f1e7bd2f938405ac2e5d41eeb7996c84dcc2/packages/charts/react-charting/src/components/CommonComponents/CartesianChart.base.tsx#L1021